AudioContextInterruptedState/explainer.md

```diff
@@ -0,0 +1,212 @@
+# AudioContext Interrupted State
+
+## Authors:
+
+- [Gabriel Santana Brito](https://github.com/gabrielsanbrito)
+- [Steve Becker](https://github.com/SteveBeckerMSFT)
+
+## Participate
+- https://github.com/WebAudio/web-audio-api/issues/2392
+
+## Introduction
+
+The [Web Audio API](https://webaudio.github.io/web-audio-api/) is widely used to add advanced audio capabilities to web applications, like web-based games and music applications. One of the API's features is the [AudioContext interface](https://webaudio.github.io/web-audio-api/#AudioContext), which represents an audio graph. An `AudioContext` can find itself in either of three states: [`"suspended"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-suspended), [`"running"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-running), or [`"closed"`](https://webaudio.github.io/web-audio-api/#dom-audiocontextstate-closed). Once an `AudioContext` is on the `"running"` state, it can only pause media playback by transitioning to the `"suspended"` when, and only when, user code calls [`suspend()`](https://webaudio.github.io/web-audio-api/#dom-audiocontext-suspend) on this `AudioContext`. However, there are situations where we might want to let the User Agent (UA) decide when to interrupt playback - e.g., the proposed [`"media-playback-while-not-visible"`](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/IframeMediaPause/iframe_media_pausing.md) permission policy and the proposed [Audio Session API](https://w3c.github.io/audio-session/); or during a phone call, where the calling application will need exclusive access to the audio hardware. To support these scenarios, we propose adding a new `"interrupted"` state to the [`AudioContextState`](https://webaudio.github.io/web-audio-api/#enumdef-audiocontextstate) enum.
+
+## Goals
+
+The main goal of this proposal is to allow the UA to be able to interrupt an `AudioContext` playback when needed, given that there are a couple of user scenarios - e.g., incoming phone calls - and proposed web API's that could make good use of this functionality.
+
+### "media-playback-while-not-visible" permission policy
+
+The ["media-playback-while-not-visible" permission policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/IframeMediaPause/iframe_media_pausing.md) allows the UA to pause media playback of unrendered iframes. However, since the Web Audio API [does not allow the UA to suspend](https://webaudio.github.io/web-audio-api/#dom-audiocontext-suspended-by-user-slot) the audio playback, the proposed permission policy has no mechanism available to pause media playback.

```

Comment:
nits: 

"allows the UA to pause media playback of unrendered iframes" --> "allows the UA to pause media playback on unrendered iframes"

"does not allow the UA to suspend the audio playback" -- > "does not allow the UA to suspend audio playback"